api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -16,6 +16,10 @@
 from pcapi.core.categories import subcategories_v2 as subcategories
 from pcapi.core.finance import utils as finance_utils
 from pcapi.core.offers import models as offers_models
+from pcapi.core.providers.constants import GTL_ID_BY_TITELIVE_MUSIC_GENRE
+from pcapi.core.providers.constants import MUSIC_SLUG_BY_GTL_ID
+from pcapi.core.providers.constants import MUSIC_SLUG_TO_GTL_ID
+from pcapi.core.providers.constants import TITELIVE_MUSIC_GENRES_BY_GTL_ID
```

Comment:
Un peu comme pour les music_types (juste en dessous) on pourrait faire un 

```suggestion
from pcapi.core.providers import constants as providers_constants
```

puis utiliser `providers_constants.GTL_ID_BY_TITELIVE_MUSIC_GENRE` etc.